pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -68,6 +69,17 @@ func (ct *connTrackSystem) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connect
 		klog.Errorf("Error when dumping flows from conntrack: %v", err)
 		return nil, err
 	}
+
+	// Update the antrea_agent_conntrack_max_connection_count prometheus metric
+	maxConns, err := sysctl.GetSysctlNet("nf_conntrack_max")
+	if err != nil {
+		klog.Errorf("Error when getting nf_conntrack_max: %v", err)
+		return nil, err
+	}
```

Comment:
I suggest the following for symmetry with the userspace case and so that we get a bit more context in the log message:
```go
getMaxConnectionsInConnTrack := func() (int, err) {
    maxConns, err := sysctl.GetSysctlNet("nf_conntrack_max")
    if err != nil {
        return 0, fmt.Errorf("rrror when getting nf_conntrack_max: %v", err)
    }
    return maxConns, nil
}

maxConns, err := getMaxConnectionsInConnTrack()
if err != nil {
    klog.Errorf("Error when getting max connections in conntrack: %v", err)
    return nil, err
}
```

Reply:
Thanks, done.